### ISSUE TITLE
Flatpak not launching through Steam

### ISSUE THREAD

**Issue by Dimolade** [vinegarhq/sober#532]
### Acknowledgement of checking existent issues

- [X] I have checked if my issue has already been reported, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

So im on Steam Deck (Which is based on Arch Linux) and ive been trying to get Sober to run but it just cant, only the installation works, then when i try to launch it after Installing, it just doesnt launch. Steam says "Stop" for a few frames but then goes back to the Green Play Button.

**Comment by Sweeistaken** [comment]
Are you able to run it in Desktop mode, what does it show if you use 
`flatpak run org.vinegarhq.Sober`
in the terminal?

**Comment by Dimolade** [comment]
> Are you able to run it in Desktop mode, what does it show if you use `flatpak run org.vinegarhq.Sober` in the terminal?

Yes Desktop Mode works fine, but i would also like to play it through Steam, which doesnt work. Also making a desktop file which launches Sober doesnt work through Steam.
